src/main/dsl/Literal.kt

```diff
@@ -1,9 +1,9 @@
 package dsl
 
-class Literal(private val characters : String) : Group() {
+class Literal(private val characters : String) : Terminal() {
     
     override fun render(builder: StringBuilder) {
```

Comment:
I'm hoping we can be consistent with most of Java/Kotlin and just use the built-in toString() method instead. I know the HTML DSL example uses render(), but they were dealing with HTML code - in the future I'm hoping to have a toRegex() method that converts our DSL objects to proper native Regex objects.